### ISSUE TITLE
Support Protobuf 27 in Python package

### ISSUE THREAD

**Issue by mering** [tensorflow/tensorflow#73920]
### Issue type

Build/Install

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

source

### TensorFlow version

2.17.1

### Custom code

No

### Current behavior?

Maximum supported Protobuf version in Python is 4.x.x where the latest one is Protobuf 26:
https://github.com/tensorflow/tensorflow/blob/84c21ad90fd67e7b039f44402a08c378d17f5b72/tensorflow/tools/pip_package/setup.py#L94

Please support Protobuf 27 which is 5.x.x on pypi.org.

### Standalone code to reproduce the issue

```shell
Use the following requirements.txt file:

protobuf==5.27.1
tensorflow==2.17.0
```

It fails to resolve dependencies because of the above mentioned requirement.
```


### Relevant log output

_No response_

**Comment by Venkat6871** [comment]
@learning-to-play , @belitskiy .

**Comment by mering** [comment]
Fixed in #76468
